jobHandler/injected/run.py

```
@@ -0,0 +1,200 @@
+def aspAdliVarLog(val, varid):
+    try:
+        val = json.dumps(val, default=lambda o: o.__dict__)
+    except:
+        pass
+    logger.info(f'# {varid} {val}')
+
+def aspTraceLog(traceType, value):
+    try:
+        value = json.dumps(value, default=lambda o: o.__dict__)
+    except:
+        pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)